scripts/db_migrations.py

```diff
@@ -92,19 +92,33 @@ def moveAuthInformationToProperties(self, resources_container_name):
 
         for item in resources_container.query_items(query='SELECT * FROM c', enable_cross_partition_query=True):
             template_version = semantic_version.Version(item["templateVersion"])
-            if (template_version < semantic_version.Version('2.5.0') and "authInformation" in item):
-                print(f'Found workspace {item["id"]} that needs migrating')
+            updated = False
+            if (template_version < semantic_version.Version('0.2.9') and item["templateName"] == "tre-workspace-base"):
 
                 # Rename app_id to be client_id
-                item["properties"]["client_id"] = item["properties"]["app_id"]
-                del item["properties"]["app_id"]
-                del item["authInformation"]["app_id"]
+                if "app_id" in item["properties"]:
+                    item["properties"]["client_id"] = item["properties"]["app_id"]
+                    del item["properties"]["app_id"]
+                    updated = True
 
-                # merge authInformation into properties
-                item["properties"] = {**item["authInformation"], **item["properties"]}
-                del item["authInformation"]
-                resources_container.upsert_item(item)
-                print(f'Upgraded authentication info for workspace id {item["id"]}')
+                if "scope_id" not in item["properties"]:
+                    updated = True
+                    item["properties"]["scope_id"] = item["properties"]["client_id"]
+
+                if "authInformation" in item:
+                    print(f'Upgrading authInformation in workspace {item["id"]}')
```

Comment:
where is this printed?
is this script still run interactively?

Reply:
This is run from the make file. There is a separate api version of this that logs

Reply:
I  have added #1824 to tidy this up